src/components/Task/Modal/Form/Inputs/ResultAutocomplete/ResultAutocomplete.test.tsx

```diff
@@ -84,4 +84,11 @@ describe('ResultSelect', () => {
       phaseData,
     });
   });
+  it('preselects a result when only one result is available', () => {
+    const { getByRole } = render(
+      <Components availableResults={[ResultEnum.Attempted]} />,
+    );
+    expect(getByRole('combobox', { name: 'Result' })).toBeInTheDocument();
+    expect(getByRole('combobox', { name: 'Result' })).toHaveValue('Attempted');
```

Comment:
The `getByRole` for the value check is enough to also assert that it is in the document.

```suggestion
    expect(getByRole('combobox', { name: 'Result' })).toHaveValue('Attempted');
```